packages/run/index.d.ts

```diff
@@ -0,0 +1,13 @@
+import { Plugin } from 'rollup'
+
+interface RollupRunOptions {
```

Comment:
The options are all equivalent to `import('child_process').ForkOptions`. It might be better to just use that type.